bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs

```diff
@@ -0,0 +1,353 @@
+        ----------
+        col: str
+            Name of column to read
+
+        Returns
+        -------
+        data: numpy.ndarray
+            The requested column of data
+        """
+        if self.template_num == None:
+            raise ValueError('You must call set_template to first pick the '
+                             'template to read data from')
+        data = self.get_data(col, self.template_num)
+        data = data[self.keep] if self.valid else data
+        return data
+
+logging.info('Starting...')
+
+num_templates = len(h5py.File(args.template_bank, "r")['template_hash'])
+tmin, tmax = parse_template_range(num_templates, args.template_fraction_range)
+logging.info('Analyzing template %s - %s' % (tmin, tmax-1))
+
+# Create dictionary for trigger files
+trigs={}
+for i in range(len(args.trigger_files)):
+    logging.info('Opening trigger file %s: %s' % (i,args.trigger_files[i]))
+    trigs[ReadByTemplate(args.trigger_files[i]).ifo]=ReadByTemplate(args.trigger_files[i],
+                                               args.template_bank,
+                                               args.segment_name,
+                                               args.veto_files)
+
+coinc_segs = trigs[ReadByTemplate(args.trigger_files[0]).ifo].segs
+for ifo in trigs:
+    coinc_segs = (coinc_segs & trigs[ifo].segs).coalesce()
+
+
+for ifo in trigs:
+    trigs[ifo].segs = coinc_segs
+    trigs[ifo].valid = veto.segments_to_start_end(trigs[ifo].segs)
+
+# initialize a Stat class instance to calculate the coinc ranking statistic
+rank_method = stat.get_statistic(args.ranking_statistic)(args.statistic_files)
+if args.use_maxalpha:
+    rank_method.use_alphamax()
+detectors={}
+for ifo in trigs:
+    detectors[ifo] = detector.Detector(trigs[ifo].ifo)
+
+# Sanity check, time slide interval should be larger than twice the
+# Earth crossing time, which is approximately 0.085 seconds.
+if 0.085 >= args.timeslide_interval and args.timeslide_interval is not None:
+    raise parser.error("The time slide interwal should be larger "
+                       "than twice the Earth crossing time.")
+
+# slide = 0 means don't do timeslides
+if args.timeslide_interval is None:
+    args.timeslide_interval = 0
+
+data = {'stat':[], 'decimation_factor':[], 'timeslide_id':[], 'template_id':[]}
+
+for tnum in range(tmin, tmax):
+    tids={}
+    for ifo in trigs:
+        tids[ifo]=trigs[ifo].set_template(tnum)
+
+    for ifo in tids:
+        if len(tids[ifo] == 0):
+            continue
+
+    times={}
+    for ifo in trigs:
+        times[ifo]=trigs[ifo]['end_time']
+    logging.info('Trigs for template %s, '% (tnum))
+    for ifo in times:
+        logging.info('%s:%s' % (ifo, len(times[ifo])))
+
+    ids, slide = coinc.time_multi_coincidence(times, args.timeslide_interval,
+                                              args.coinc_threshold,
+                                              trigs[ReadByTemplate(args.trigger_files[0]).ifo].ifo,
```

Comment:
Doesn't this line seem a bit redundant? 'trigs' is already a dict of ReadByTemplate instances and all one needs to pass here is the ifo name.

Reply:
The 'fixed' and 'pivot' ifos should be specified directly on the command line. Otherwise, it is dependent on file order, which is a bit non-obvious and also not completely guaranteed.